crates/circuit/src/dag_circuit.rs

```diff
@@ -4768,6 +4768,12 @@ def _format(operand):
 }
 
 impl DAGCircuit {
+    /// Returns an immutable view of the qubit io map
+    #[inline(always)]
+    pub fn qubit_io_map(&self) -> &[[NodeIndex; 2]] {
+        &self.qubit_io_map
+    }
```

Comment:
Do you think it'd make sense to have one of these getters for the `clbit_io_map` too? I assume it's not needed now, but for the sake of consistency?

Reply:
Probably, in practice I don't think we do that very often in the transpiler. But having the method for consistency would be good.